src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -52,6 +52,7 @@
               :showErrors="false"
               :businessLookup="initialBusinessLookupObject"
               :BusinessLookupServices="BusinessLookupServices"
+              legalTypes="BC,BEN,CC,ULC,A"
```

Comment:
This is probably fine for now since we'll need to do additional work to support other legal types, I think.

Reply:
Yup, I asked Mihai last time and what to search exactly for now:
![mihai answer](https://github.com/bcgov/business-create-ui/assets/122301442/453d2fe1-f61c-4037-81f2-8773d16e986b)

Reply:
Yeah, but I meant there's little advantage to parameterizing this (or FF).